pages/linux/losetup.md

```diff
@@ -0,0 +1,19 @@
+# losetup
+
+> Set up and control loop devices.
+
+- List loop devices with detailed info:
+
+`losetup -a`
+
+- Attach a file to a given loop device:
+
+`sudo losetup /dev/{{loop}} /{{path/to/<file>}}`
+
+- Detach All loop devices:
```

Comment:
Thank you, you can mount iso,img and many others. 
Also, you can make images with dd and use them with losetup.